crates/spidermonkey-embedding-splicer/src/bindgen.rs

```diff
@@ -385,6 +385,11 @@ impl JsBindgen<'_> {
                         if let Some(name) = iface.name.as_ref() {
                             let camel_case_name = name.to_lower_camel_case();
                             if !guest_exports.contains(&camel_case_name) {
+                                if name == "incoming-handler"
```

Comment:
Let's just add a brief comment here as well to ensure this is properly motivated.

Reply:
added.